### ISSUE TITLE
why the checkpoints is biger

### ISSUE THREAD

**Issue by chongxian** [PixArt-alpha/PixArt-sigma#34]
hello,when I use the PixArt Training Tutorial  to train model, the origin checkpoints "PixArt-Sigma-XL-2-256x256.pth"only has 2.9GB, but my train result has 4.19GB, how to use the trained result ? thanks for your reply

**Comment by ApolloRay** [comment]
> hello,when I use the PixArt Training Tutorial to train model, the origin checkpoints "PixArt-Sigma-XL-2-256x256.pth"only has 2.9GB, but my train result has 4.19GB, how to use the trained result ? thanks for your reply

fp16 and fp32 ? maybe

**Comment by lawrence-cj** [comment]
Maybe the optimizer's parameter is also saved?

**Comment by chongxian** [comment]
> Maybe the optimizer's parameter is also saved?
I follow this command,but get the 4.19GB
![image](https://github.com/PixArt-alpha/PixArt-sigma/assets/18145013/7026b685-1436-4bd8-952b-d2a2f7d5e792)

**Comment by chongxian** [comment]
> > hello,when I use the PixArt Training Tutorial to train model, the origin checkpoints "PixArt-Sigma-XL-2-256x256.pth"only has 2.9GB, but my train result has 4.19GB, how to use the trained result ? thanks for your reply
> 
> fp16 and fp32 ? maybe

fp16

**Comment by lawrence-cj** [comment]
> Maybe the optimizer's parameter is also saved?
No?

**Comment by chongxian** [comment]
> > Maybe the optimizer's parameter is also saved?
> 
> No?

I am not quite sure，I only modify PixArt_sigma_xl2_img256_internal.py context about the data file,how can I do to get the right checkpoints?
![image](https://github.com/PixArt-alpha/PixArt-sigma/assets/18145013/2c3dfe98-79c6-49b2-8204-8e9db0cf7610)

**Comment by chongxian** [comment]
> > Maybe the optimizer's parameter is also saved?
> 
> No?

yes, include epoch ,optimizer,scheduler

**Comment by chongxian** [comment]
> > Maybe the optimizer's parameter is also saved?
> 
> No?

but I can use the comfyUi to load this checkpoints, report the error about loading state_dict for PixArtMS,but can load PixArt-Sigma-XL-2-512-MS.pth and others
![image](https://github.com/PixArt-alpha/PixArt-sigma/assets/18145013/bfffb0e9-52ad-44f9-b0e3-37b1bbb08283)
